cmd/agent/main.go

```diff
@@ -352,9 +361,9 @@ func main() {
 			} else {
 				prometheus.MustRegister(netdev)
 			}
-			http.Handle("/metrics", mahttp.ValidationMiddleware(promHandler))
+			mux.Handle("/metrics", mahttp.ValidationMiddleware(promHandler))
```

Comment:
I'm think that it may not be a good idea to export both agent (intrinsic) metrics and influx collected metrics under the same endpoint, and differentiate on the individual metric names. It's certainly one way to do it, but for someone who's only interested in solana node data, vs the agent data, it will pollute their timeseries database with agent metrics (and perhaps, in some cases, slightly complicate their processing).

Reply:
Done. Influx PEF metrics are exposed under the reverse proxy's `/metrics` endpoint.